src/utils/tutorials.js

```diff
@@ -234,28 +234,4 @@ export function isLessonPassed (tutorial, lesson) {
   return !!localStorage[`passed/${tutorial.url}/${lesson.formattedId}`]
 }
 
```

Comment:
If these redirects are needed, do you plan to land the netlify config to recreate them before merging this PR?

Reply:
This feature is not being used still, but it may be in the future.
Meaning, only when we have `redirectUrls` in any of the (tutorials](https://github.com/ProtoSchool/protoschool.github.io/blob/main/src/static/tutorials.json#L4) will this feature be used.

But regarding the issue #520, I've changed its goal to actually include client side redirects to be used as fallback while the server configuration is not set yet. This seems like a good approach to not rely on the server config for the redirects to work, but instead, rely on them only for the SEO.
I'm working on a PR for this already.

So the removal of this code here will not break anything.

Reply:
PR to add pre-rendered pages for redirects added here: https://github.com/ProtoSchool/protoschool.github.io/pull/525